.github/workflows/ci_windows.yaml

```diff
@@ -55,9 +67,33 @@ jobs:
             --clean-after-build `
             $packages
 
-      # this is necessary to correctly find and use cl.exe with the Ninja generator for now
-      - name: Setup devcmd
-        uses: ilammy/msvc-dev-cmd@v1
+      - name: Install boost
+        run: |
+          aria2c `
+            "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.7z" `
```

Comment:
Wouldn't it be better to declare the boost version somewhere above?

Reply:
>Wouldn't it be better to declare the boost version somewhere above?

@glassez Do you mean something like below:

```
  id: install-boost
  with:
    # REQUIRED: Specify the required boost version
    # A list of supported versions can be found here: 
    # https://github.com/actions/boost-versions/blob/main/versions-manifest.json
    boost_version: 1.73.0
    # OPTIONAL: Specify a custon install location
    boost_install_dir: C:\some_directory
```

Reference:
https://github.com/MarkusJx/install-boost or perhaps even https://github.com/egor-tensin/build-boost

Reply:
Or this:
```
BOOST_VERSION = 1.76.0
BOOST_SOURCE = boost_$(subst .,_,$(BOOST_VERSION)).tar.bz2
BOOST_SITE = https://boostorg.jfrog.io/artifactory/main/release/$(BOOST_VERSION)/source
```

Reply:
>Wouldn't it be better to declare the boost version somewhere above?

Although the URL is relatively stable, it still might change over time. I can add an comment for version if that is what you need.

Reply:
I meant to make it so that changing the Boost version does not require changing the URL string inside the download script, but just some variable. However, it's not that important - just a little convenience.